src/osd/OSD.cc

```diff
@@ -2570,6 +2570,18 @@ float OSD::get_osd_recovery_sleep()
     return cct->_conf->osd_recovery_sleep_hdd;
 }
 
+float OSD::get_osd_snap_trim_sleep()
+{
+  float osd_snap_trim_sleep = cct->_conf.get_val<double>("osd_snap_trim_sleep");
```

Comment:
@smithfarm make check is complaining about this, you can just follow https://github.com/ceph/ceph/blob/luminous/src/osd/OSD.cc#L2568